### PR TITLE
pipeline: inputs: kubernetes-events: Fix typo

### PR DESCRIPTION
**Pull request by sh-cho**
prece'n'dence -> precedence

### CODE REVIEW

File: pipeline/inputs/kubernetes-events.md [hunk 1]
```diff
@@ -50,4 +50,4 @@ In the following configuration file, the input plugin *kubernetes_events* collec
 ```
 
 ### Event Timestamp
-Event timestamp will be created from the first existing field in the following order of precendence: lastTimestamp, firstTimestamp, metadata.creationTimestamp
+Event timestamp will be created from the first existing field in the following order of precedence: lastTimestamp, firstTimestamp, metadata.creationTimestamp
```
Comment (cnorris-cs) [inline pipeline/inputs/kubernetes-events.md:53]: Added line wrapping, removed passive voice, and turned the list into an ordered list to better indicate the order of precedence.

```suggestion

Event timestamps are created from the first existing field, based on the following
order of precedence:

1. `lastTimestamp`
1. `firstTimestamp`
1. `metadata.creationTimestamp`
```

Comment (cnorris-cs) [reply pipeline/inputs/kubernetes-events.md:53]: As a note, Markdown will turn the "1"s into "1, 2, 3".

**Review by cnorris-cs — APPROVED** [review]
I left a suggestion, but I've added my approval based on that to keep this moving.

File: pipeline/inputs/kubernetes-events.md (hunk 1 above)
Comment (sh-cho) [reply pipeline/inputs/kubernetes-events.md]: Thanks!

**Pull request merged**